content/kotlin/concepts/strings/terms/dropLast/dropLast.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: 'dropLast()'
+Description: 'Returns a string with n characters removed.'
+Subject:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Data Types'
+  - 'Characters'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The Kotlin method, **`.dropLast()`** returns a string with the last `(n)` characters removed.
+
+## Syntax
+
+```pseudo
+value string = "Example String"
+println(string.dropLast())
+```
+
+### Special Cases
+
+`IllegalArguementException` if `n` is negative.
+
+## Example
+
+The example below creates a string, `string`, then uses the `.dropLast()` method to return astring with the last character and a string the last five characters removed.
+
+```kotlin
+value string = "Example String"
+println(string.dropLast())
+println(string.dropLast(5))
```

Comment:
The code is not executing properly. Please correct the code and remember to write it inside the main function otherwise it'll throw an error.